parquet-common/src/test/java/org/apache/parquet/TestLog.java

```diff
@@ -21,11 +21,16 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class TestLog {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(TestLog.class);
+
   @Test
   public void test() {
```

Comment:
I don't think this class is needed any more. It just validates that we don't release code with Log.DEBUG set to true. That no longer matters.